src/pages/video-page/VideoPage.jsx

```diff
@@ -44,8 +45,6 @@ export function VideoPage() {
     window.scrollTo(0, 0);
```

Comment:
Suggestion: You could extract out the window scroll up functionality to a component and run it in useEffect same as you have used here with pathname dependency